Regression/WarpX-tests.ini

```diff
@@ -2137,7 +2137,7 @@ analysisRoutine = Examples/analysis_default_regression.py
 [LaserAccelerationBoost]
 buildDir = .
 inputFile = Examples/Physics_applications/laser_acceleration/inputs_2d_boost
-runtime_params = warpx.do_dynamic_scheduling=0 warpx.serialize_initial_conditions=1 amr.n_cell=64 512 max_step=20
+runtime_params = warpx.do_dynamic_scheduling=0 warpx.serialize_initial_conditions=1 amr.n_cell=64 512 max_step=300
```

Comment:
Note: `max_step` increased to `300` to make sure that we finish writing all three snapshots (newly added to the test), thus avoiding the warning implemented in this PR. New runtime on Azure is about 6 s, so still fast enough for CI.

Reply:
Great!  Thanks, Edoardo